src/libawkward/forth/ForthOutputBuffer.cpp

```diff
@@ -196,13 +196,13 @@ namespace awkward {
 
   template <typename OUT>
   void
-  ForthOutputBufferOf<OUT>::write_one_bool(bool value, bool byteswap) noexcept {
+  ForthOutputBufferOf<OUT>::write_one_bool(bool value, bool) noexcept {
```

Comment:
These `byteswap` arguments _should_ stay here for symmetry with the other methods. They're not used in a few of the methods (one method for each data type) because some data types have only one byte, and therefore nothing to swap.